pkg/solana/chain_reader.go

```diff
@@ -0,0 +1,76 @@
+package solana
+
+import (
+	"context"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/logger"
+	"github.com/smartcontractkit/chainlink-common/pkg/services"
+	commonservices "github.com/smartcontractkit/chainlink-common/pkg/services"
+	"github.com/smartcontractkit/chainlink-common/pkg/types"
+	commontypes "github.com/smartcontractkit/chainlink-common/pkg/types"
```

Comment:
You have `"github.com/smartcontractkit/chainlink-common/pkg/types"` imported twice here, either one is fine but we don't need both.